puppet/modules/web/files/deploy-yumrepo.sh

```diff
@@ -23,7 +23,10 @@ do_rsync() {
 	fi
 
 	rsync "${opts[@]}" --log-file-format 'CHANGED %f' "${REPO_SOURCE_RPM}/*" "${REPO_INSTANCE_PATH}/"
-	rsync "${opts[@]}" --log-file-format 'CHANGED source/%f' "${REPO_SOURCE_SRPM}/" "${REPO_INSTANCE_PATH}/source/"
+
+	if [[ -n "$REPO_SOURCE_RPM" ]]; then
```

Comment:
If curious, the staging from Copr already sets up the structure to match how it is in production, e.g. (https://stagingyum.theforeman.org/foreman/nightly/el8/):

```
[DIR]	source/	2023-08-29 15:15	-	 
[DIR]	x86_64/	2023-08-29 15:15	-	 
```

Reply:
I think you intended to check the SRPM var, because both branches set RPM.
```suggestion
	if [[ -n "$REPO_SOURCE_SRPM" ]]; then
```

Reply:
Oy, yes. I had to stare hard at that. Updated.